src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig

```diff
@@ -23,11 +23,11 @@
     <div class="input-group">
         {% set append = money_pattern starts with '{{' %}
         {% if not append %}
-            <span class="input-group-addon">{{ money_pattern|replace({ '{{ widget }}':''}) }}</span>
+            <span class="input-group-addon">{{ money_pattern|replace({ '{{ widget }}':''})|raw }}</span>
```

Comment:
now assuming `money_pattern` is a safe value. I think it's truly intended looking at `money_widget.html.php` and `form_div_layout.html.twig`.